sdk/schemaregistry/schema-registry-avro/tsconfig.json

```diff
@@ -1,13 +1,20 @@
 {
   "extends": "../../../tsconfig",
   "compilerOptions": {
-    "outDir": "./dist-esm",
-    "declarationDir": "./types",
     "paths": {
-      "@azure/schema-registry-avro": ["./src/index"]
+      "@azure/schema-registry-avro": ["./src/index.js"]
     },
-    "lib": ["dom"],
-    "skipLibCheck": true
+    "skipLibCheck": true,
```

Comment:
Is this needed? I hope not!

Reply:
It's not needed, seems to have been added during merge conflict

Reply:
hmm, it was added in main to work around LRU issue with typescript 5.6, Seems the issue is no longer there?

https://github.com/isaacs/node-lru-cache/issues/348